requirements/prod.txt

```diff
@@ -13,6 +13,7 @@ netaddr
 numpy
 msgpack-numpy-opentensor==0.5.0
 nest_asyncio
+packaging==24.0
```

Comment:
This dependency was already present in the project, but I couldn't tell which package introduced it, so decided to pin it explicitly

Reply:
please do not pin it!

You were right in adding all directly used packages here, but pinning stuff in library is a bad idea generally, as whatever project may want to install it may have conflicts if 2+ libraries had similar idea.

```suggestion
packaging
```